packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -337,7 +341,7 @@ describe("Data Store Context Tests", () => {
             it("can correctly initialize and generate attributes", async () => {
                 dataStoreAttributes = {
                     pkg: JSON.stringify(["TestDataStore1"]),
-                    snapshotFormatVersion: "0.1",
+                    summaryFormatVersion: 2,
                     isRootDataStore: true,
```

Comment:
I was thinking of tests where these attributes in the initial snapshot for the data store has `disableIsolatedChannels` as true / false. And make sure we can read and process the snapshot accordingly.

Reply:
I've made changes, let me know if this is what you're looking for. It now runs the initialization tests on all 8x combinations.

Read x4: v0, v1, v2, v2 + disableIsolatedChannels
Write x2: v1, v2

Reply:
This looks good!